trackidledetector-processor/src/main/resources/com/esri/geoevent/processor/trackidledetector-processor.properties

```diff
@@ -1,26 +1,46 @@
 # Processor Definition
 PROCESSOR_LABEL=Track Idle Detector
-PROCESSOR_DESC=Detects non-movement of a Track beyond a specified period of time.
+PROCESSOR_DESC=Release ${project.release}: Detects non-movement of a Track beyond a specified period of time.
```

Comment:
Release???

Reply:
Version is the GeoEvent SDK version that the bundle was compiled against. And we don't change that if a bundle is enhanced or has a bug fixed, so we need another way to indicate a bundle's version. So I added the project.release property to indicate a bundle's release (aka version) that is independant of the SDK version.

Reply:
See the following for example of how the release is used:

[Track Idle Detector Processor Gallery Item page](https://geoevent.maps.arcgis.com/home/item.html?id=5d8e3446736d4df299c7c96bc275d561)